src/components/Tooltip/Tooltip.js

```diff
@@ -384,6 +398,37 @@ export default class Tooltip extends Component {
           'aria-owns': tooltipId,
         };
 
+    const finalIcon = IconCustomElement ? (
```

Comment:
Can we assign `Information` to `IconCustomElement` if it's not given and `componentsX` is truthy, so we can merge the block here and the next block?

Reply:
I have done something like this 

```js 
    const IconCustomElement = renderIcon || componentsX && Information;
    
    const finalIcon = IconCustomElement ? (
      <IconCustomElement
        name={iconName}
        aria-labelledby={triggerId}
        aria-label={iconDescription}
        ref={node => {
          this.triggerEl = node;
        }}
      />
    ) : (
      <Icon
        icon={!icon && !iconName ? iconInfoGlyph : icon}
        name={iconName}
        description={iconDescription}
        iconTitle={iconTitle}
        iconRef={node => {
          this.triggerEl = node;
        }}
      />
    );
```

But maybe a proper name instead of IconCustomElement, considering this change, could be `IconCustomOrInformation`.